packages/hdwallet-native/jest.config.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  preset: "ts-jest",
+//   testEnvironment: "node",
+  testPathIgnorePatterns: ["dist"],
+  globals: {
```

Comment:
Using a jest.config.js instead of embedding the configuration into package.json is required to pass through Uint8Array and ArrayBuffer globals, which we need to work around facebook/jest#4422.